### ISSUE TITLE
where can i find lastApplied 

### ISSUE THREAD

**Issue by xiaozhiliaoo** [ongardie/raftscope#21]
It seems not show lastApplied . where can i find it?

**Comment by ongardie** [comment]
Correct. There's no state machine in the visualization, so nothing is "applying" the committed entries, so it's not meaningful to keep track of the last one that was applied. In a normal implementation, the state machine applies committed entries as quickly as possible, and when it does so, it increments lastApplied accordingly.
